test/sql/drop_chunks.sql

```diff
@@ -70,6 +71,20 @@ WHERE h.schema_name = 'public' AND (h.table_name = 'drop_chunk_test1' OR h.table
 
 \dt "_timescaledb_internal".*
 
+-- 2,147,483,647 is the largest int so this tests that BIGINTs work
+SELECT drop_chunks(2147483648, 'drop_chunk_test3');
```

Comment:
Shouldn't we actually insert some data into `drop_chunk_test3` before dropping chunks so we can test that it actually does something?

Reply:
Bah sorry forgot that when I redid.